bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/MenuHeader.java

```diff
@@ -21,21 +20,14 @@ public class MenuHeader extends AbstractLink {
     public MenuHeader(final IModel<String> label) {
         super(ButtonList.getButtonMarkupId());
 
-        setRenderBodyOnly(true);
         setBody(label);
     }
 
     @Override
-    protected void onInitialize() {
-        super.onInitialize();
-
-        ListItemCssClassHelper.onInitialize(this, "dropdown-header");
-    }
-
-    @Override
-    protected void onConfigure() {
-        super.onConfigure();
-
-        ListItemCssClassHelper.onConfigure(this, "dropdown-header");
+    protected void onComponentTag(final ComponentTag tag) {
```

Comment:
This led to broken build: https://github.com/l0rdn1kk0n/wicket-bootstrap/actions/runs/3150041159/jobs/5122365805

Reply:
Thanks, I edited it via GitHub editor and not in an IDE, so did not spot it!